### PR TITLE
Update Dependencies 

### PR DESCRIPTION
**Pull request by utkarsh006**
- Fixes Issue #18 
- App build is successful.

### CODE REVIEW

**Comment by utkarsh006** [comment]
@hieuwu Kindly Review.

File: local.properties [hunk 1]
```diff
@@ -4,8 +4,8 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-#Tue Oct 10 04:51:12 IST 2023
+#Sun Oct 15 13:05:21 IST 2023
 API_KEY=YOUR_SUPABASE_KEY
 SECRET=YOUR_SUPABASE_SECRET
 SUPABASE_URL=YOUR_SUPABASE_URL
-sdk.dir=/Users/hieuvu/Library/Android/sdk
+sdk.dir=C\:\\Users\\HP\\AppData\\Local\\Android\\Sdk
```
Comment (hieuwu) [inline local.properties:11]: Please revert this change

File: app/build.gradle [hunk 2]
```diff
@@ -134,19 +133,11 @@ dependencies {
...
+tasks.register('wrapper', Wrapper) {
     gradleVersion = '7.5'
 }
 
 detekt {
     config = files("${project.rootDir.absolutePath}/config/detekt/detekt.yml")
     buildUponDefaultConfig = true
 }
-
-ktlint {
```
Comment (hieuwu) [inline app/build.gradle:146]: Please revert this change

File: app/build.gradle [hunk 3]
```diff
@@ -118,13 +117,13 @@ dependencies {
     implementation("androidx.hilt:hilt-navigation-compose:1.0.0")
     implementation "com.github.bumptech.glide:compose:1.0.0-alpha.1"
     implementation "androidx.compose.material3:material3:1.1.1"
-    testImplementation("org.mockito:mockito-core:2.1.0")
+    testImplementation("org.mockito:mockito-core:5.3.1")
```
Comment (hieuwu) [inline app/build.gradle:120]: Please revert changes related to Supabase.

File: app/build.gradle [hunk 4]
```diff
@@ -40,12 +41,12 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_11
-        targetCompatibility JavaVersion.VERSION_11
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
```
Comment (hieuwu) [inline app/build.gradle:45]: Please revert this to as current java version is 12 but new one is java 8.

**Review by hieuwu — APPROVED** [review]

File: local.properties (hunk 1 above)
Comment (utkarsh006) [reply local.properties:11]: @hieuwu This change is auto generated

File: app/build.gradle (hunk 2 above)
Comment (utkarsh006) [reply app/build.gradle:146]: @hieuwu Done

File: app/build.gradle (hunk 4 above)
Comment (utkarsh006) [reply app/build.gradle:45]: @hieuwu Reverted to Java12

**Comment by utkarsh006** [comment]
@hieuwu Review now
